.gitignore

```diff
@@ -3,3 +3,5 @@
 Homestead.yaml
 .env
 .env.test
+_ide_helper.php
```

Comment:
No need to add files here for your local environment; you can add those instead to your global git ignore.